pom.xml

```diff
@@ -211,9 +211,8 @@
         <version.org.wildfly.openssl.wildfly-openssl-solaris-x86_64>${version.org.wildfly.openssl.natives}</version.org.wildfly.openssl.wildfly-openssl-solaris-x86_64>
         <version.org.wildfly.openssl.wildfly-openssl-windows-i386>${version.org.wildfly.openssl.natives}</version.org.wildfly.openssl.wildfly-openssl-windows-i386>
         <version.org.wildfly.openssl.wildfly-openssl-windows-x86_64>${version.org.wildfly.openssl.natives}</version.org.wildfly.openssl.wildfly-openssl-windows-x86_64>
-        <version.org.wildfly.security.elytron>1.8.0.Final</version.org.wildfly.security.elytron>
+        <version.org.wildfly.security.elytron>1.9.0.CR2</version.org.wildfly.security.elytron>
         <version.org.wildfly.security.elytron-web>1.4.0.Final</version.org.wildfly.security.elytron-web>
```

Comment:
Elytron Web continue to get its own release cycle, right?

Reply:
Once this is in I will create a new tag of Elytron Web that depends on the modularised version - however yes Elytron web will continue to be independent. 

The reason we keep that one independent is it depends on one or more HTTP servers so we don't want to tie all those dependencies to the main Elytron project.